template/addons/next-auth/api-handler.ts

```diff
@@ -4,6 +4,13 @@ import CredentialsProvider from "next-auth/providers/credentials";
 import { env } from "../../../server/env";
 
 export const authOptions: NextAuthOptions = {
+  // Include user.id on session
```

Comment:
Why do we have a separate prisma and non-prisma version? These look identical

Reply:
`template/addons/next-auth/api-handler.ts` and `template/addons/next-auth/api-handler-prisma.ts` are different. 
The latter sets the Adapter as `PrismaAdapter`

https://github.com/t3-oss/create-t3-app/blob/b2d542a44eed0aa699fe91e52f25f6ac68a0021c/template/addons/next-auth/api-handler-prisma.ts#L12